supplementary_style_guide/style_guidelines/release-notes.adoc

```diff
@@ -63,108 +63,433 @@ Introduce each release note with a heading that summarizes the release note. Thi
-Workaround: Rerun the deployment command, or use a local container image registry on the undercloud.
+Description - describe the removed feature
+Consequence - describe the recommended replacement, if applicable
 ----
 
-=== Technology Preview
+.Removed feature release note text
+====
+<feature> is removed::
+The _<feature>_, which _<purpose>_, is removed and is no longer supported. You can _<purpose>_ by using _<alternative>_ instead.
++
+TICKET-REFERENCE
+====
 
-For guidance and the template text to use for Technology Preview features, see the xref:technology-preview-guidance[Technology Preview] section.
+In addition to general style, follow these guidelines:
 
-[[deprecated-and-removed-features]]
-=== Deprecated and removed features
+* If a functionality is removed in a release (for example, in RHEL 9), it must be documented as deprecated in a preceding release (RHEL 8).
+* If a small part of a feature is removed, treat that as a feature change, not a removed feature. Focus on why the change was made and what replaces the removed item. 
 
-Documenting the deprecation and removal stages of software features requires careful and precise communication.
-Highlight the following stages to users:
+.Examples of removed feature release notes
+====
+`scap-workbench` is removed::
+The `scap-workbench` package is removed in RHEL 10. The `scap-workbench` graphical utility performed configuration and vulnerability scans on a single local or remote system. As an alternative, you can scan local systems for configuration compliance by using the `oscap` command and remote systems by using the `oscap-ssh` command. For more information, see link:https://docs.redhat.com/en/documentation/red_hat_enterprise_linux/10/html/security_hardening/scanning-the-system-for-configuration-compliance#configuration-compliance-scanning[Configuration compliance scanning].
++
+RHELDOCS-19009
+====
 
-* Plan to deprecate
-* Deprecate
-* Plan to remove
-* Remove
+====
+Service Binding Operator documentation removed::
+With this release, the documentation for the Service Binding Operator (SBO) has been removed because this Operator is no longer supported.
++
+TICKET-REFERENCE
+====
 
-When alternatives to or workarounds for deprecated features are available, clearly inform users about them.
+If your product presents deprecations and removals in a table, follow the guidance for deprecation tables.
 
-==== Referring to releases in deprecation and removal notices
-In general, avoid definitive statements about specific releases, release versions, or dates for deprecation or removal.
-When possible, use the phrase "is planned for a future release" because it accounts for the possibility of changes to the planned deprecation or removal timeline.
+* Remove the entry from the table when the release for that removal is no longer fully supported. Typically that means a removal is not carried on beyond n-2 versions of the product documentation.
```

Comment:
The second sentence is a little confusing the way that it's currently worded. How about this instead? 

> Typically you do not need to include a removal in the release notes beyond n-2 versions of the product documentation.

Or maybe:
> Typically you do not need to include a removal in the release notes for more than n-2 versions of the product documentation.